implementation/clabbert/build.gradle

```diff
@@ -1,42 +1,100 @@
 plugins {
-	id 'java'
-	id 'org.springframework.boot' version '3.4.1'
-	id 'io.spring.dependency-management' version '1.1.7'
+    id 'java'
+    id 'org.springframework.boot' version '3.4.1'
+    id 'io.spring.dependency-management' version '1.1.7'
+    id("com.diffplug.spotless") version "7.0.0.BETA4"
```

Comment:
**nit:** Should we use a beta version here instead of last stable?

Reply:
It's possible, but it's just a formatter, so I'll ignore your comment (especially since in the “stable” version for some reason putting the year in copyright comment doesn't work properly)